backend/capellacollab/tools/crud.py

```diff
@@ -0,0 +1,76 @@
+# Copyright DB Netz AG and the capella-collab-manager contributors
+# SPDX-License-Identifier: Apache-2.0
+
+import typing as t
+from ensurepip import version
+
+from sqlalchemy.orm import Session
+
+from capellacollab.tools.models import Tool, Type, Version
+
+############
+### Tool ###
+############
+
```

Comment:
Please delete these comments to make it more consistent to the repository since these are nowhere else used.

Reply:
Ok but I still think that it’s useful to group the functions to ease the navigation in the file. Do you have any other suggestion than these comments?